src/Graphula.hs

```diff
@@ -191,12 +191,18 @@ data Args backend n m = Args
   , gen :: IORef QCGen
   }
 
+type role Args representational representational representational
```

Comment:
This seems like a ton of overhead with no value. Can we `-Wno-missing-role-annotations` instead? We've done that many times for new warnings, e.g. with `missing-kind-signatures`, etc.

When it comes to our warnings choices and use of `everything`, the intent is not "seek to satisfy `everything`, only opting out what you must", it's more "`everything` + opt-out is better than opt-in" -- so there's no stigma to opting out of stuff. Would be good to document this somewhere I guess.

Unless there's some big benefit to this that I'm missing?

Reply:
> make sure GHC agrees with what you thought a role was

FWIW, I have exactly 0 idea what any role of anything is at any point. What you have here is greek to me :shrug: